### DISCUSSION TITLE
Not getting any results if my query lacks an apostrophe

### DISCUSSION THREAD

**Discussion by dtlaurie** [osm-search/Nominatim#3443] [Q&A]
Hello everyone, 

I installed Nominatim 4.0 on a local network and when I try to search an address that lacks an apostrophe I get no results.

But when I do the same on your official website(4.4) it works wonderdully.

Could you help me out with that ?

Thanks a lot

Best regards,

**Comment by mtmail** [comment]
Do you have an example?

Can you install a newer version of Nominatim? It will require a full reimport of all data. Nominatim 4.0 was at least 2 years ago and in the meantime many bugs have been fixed and improvements made.

**Reply by dtlaurie** [reply]
Hi, thanks a lot for your reply !

Here's an example query that does not return any result on my local installation but works on the official website : 

[https://nominatim.openstreetmap.org/search?q=st%20sauveur%20d%20aunis&format=json](https://nominatim.openstreetmap.org/search?q=st%20sauveur%20d%20aunis&format=json)

The query string lacks an apostrophe between "d" and "aunis", on purpose.

I've thought about upgrading Nominatim on my local network but I'm getting some trouble with the docker image, is there any documentation I could use ? 

Thanks a lot

**Reply by mtmail** [reply]
I imported the area (https://www.openstreetmap.org/relation/8652) with Nominatim 4.0 and 4.4. Indeed it's fixed in version 4.4. Docker setup is a separate project, have a look at https://github.com/mediagis/nominatim-docker/ and open an issue there if you run into trouble.

**Reply by dtlaurie** [reply]
Thanks a lot for your help!
